partitioned-table.md

```diff
@@ -1662,6 +1662,101 @@ CREATE TABLE t (a varchar(20), b blob,
 ERROR 1503 (HY000): A UNIQUE INDEX must include all columns in the table's partitioning function
 ```
 
```

Comment:
It seems that L1456 also needs updates so users can know that the mentioned rules (limitations) in that line only apply to the scenarios where `tidb_enable_global_index` is not enabled.

Reply:
Added a note here https://github.com/pingcap/docs/pull/18543/files#diff-20864a53c3332130114abe4d0bd81371983d49d5ad8cc6f11efd17e06ddc5302R1458